README.md

```diff
@@ -35,15 +35,31 @@ your needs. The format is shown in the example. The simplest form would be:
 ```
 
 *Important Note:* You have to restart foreman in order to apply changes in
-`default_hostgroup.yaml`!
+`foreman_default_hostgroup.yaml`!
```

Comment:
I renamed it because the docs where out of date.  The filename is actually foreman_default_hostgroup and that is how the gem/rpm installs it.  Having the incorrect name here will confuse people and the plugin will not even function (afaik).

Since I was updating the docs I thought I would just fix a few other things that needed updating.